lea/databases.py

```diff
@@ -41,6 +44,9 @@ class DatabaseClient(typing.Protocol):
     def create_dataset(self, dataset_name: str):
         pass
 
+    def create_schema(self, TableRef: scripts.TableRef):
+        pass
```

Comment:
```suggestion
```

No need to add this to the protocol, because it's just a DuckDB method (at least for now)